cmds/cpio/pkg/types.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2013-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// basic types for cpio
+
+package cpio
+
+import (
+	"io"
+)
+
+const (
+	// bad news. These are not defined on
+	// whatever version of Go travis is running.
```

Comment:
set go version to 1.9 here? https://github.com/u-root/u-root/blob/master/.travis.yml